torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3289,6 +3289,10 @@ def error_inputs_adaptive_avg_pool2d(opinfo, device, **kwargs):
     # error inputs for output_size lesser than 0
     yield ErrorInput(SampleInput(make_arg((1, 1, 1, 1)), output_size=(-1, 0)),
                      error_regex="elements of output_size must be greater than or equal to 0")
+    
+    # error inputs for incorrect output_size dimension
+    yield ErrorInput(SampleInput(make_arg((1, 1, 1, 1)), output_size=(2, 2, 2)),
+                     error_regex="output_size must be of type int or have dimension 2")
```

Comment:
The failures are because the string needs to match the error message
```suggestion
                     error_regex="output_size must be of type int (for square output) or have dimension 2")
```